cpp/src/backends/process/model_worker.cc

```diff
@@ -15,56 +17,57 @@ namespace torchserve {
     if (socket_type == "unix") {
       socket_family = AF_UNIX;
       if (socket_name.empty()) {
-        LOG(FATAL) << "Wrong arguments passed. No socket name given.";
+        TS_LOG(FATAL, "Wrong arguments passed. No socket name given.");
       }
-      
-      std::filesystem::path s_name_path(socket_name);
-      if (std::remove(socket_name.c_str()) != 0 && std::filesystem::exists(s_name_path)) {
-        LOG(FATAL) << "socket already in use: " << socket_name;
+
+      fs::path s_name_path(socket_name);
+      if (std::remove(socket_name.c_str()) != 0 && fs::exists(s_name_path)) {
+        TS_LOGF(FATAL, "socket already in use: {}", socket_name);
       }
       socket_name_ = socket_name;
     } else if (socket_type == "tcp") {
       if (host_addr.empty()) {
         socket_name_ = "127.0.0.1";
       } else {
         socket_name_ = host_addr;
-        if (port_num.empty())
-            LOG(FATAL) << "Wrong arguments passed. No socket port given.";
+        if (port_num.empty()) {
+          TS_LOG(FATAL, "Wrong arguments passed. No socket port given.");
+        }
         port_ = htons(stoi(port_num));
       }
     } else {
-        LOG(FATAL) << "Incomplete data provided";
+      TS_LOG(FATAL, "Incomplete data provided");
     }
 
-    LOG(INFO) << "Listening on port: " << socket_name;
+    TS_LOGF(INFO, "Listening on port: {}", socket_name);
```

Comment:
Please refer https://github.com/pytorch/serve/blob/master/ts/model_service_worker.py#L172. this log is used to verify backend start successfully, (see https://github.com/pytorch/serve/blob/master/frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerLifeCycle.java#L249)

Reply:
That log statement is included at L87.